client/components/result-card.jsx

```diff
@@ -16,46 +17,58 @@ export default function ResultCard(props) {
       textDecoration: 'none'
     },
     background: {
-      height: '220px',
+      height: '250px',
       background: `url(${result.image_url}) no-repeat center`,
       boxShadow: 'inset 0 0 0 100vw rgb(0 0 0 / 60%)',
       backgroundSize: 'cover',
       borderRadius: '10px'
-    },
-    name: {
-      fontSize: '1.5rem'
     }
   };
   const miles = distance * METERS_TO_MILES;
   const displayPhoneIcon = phone
-    ? <><Card.Text as='span' className='phone-icon fas fa-phone' /><br /></>
+    ? (
+      <>
+        <Card.Text
+          as='span'
+          className='phone-icon fas fa-phone' />
+        <br />
+      </>
+      )
     : null;
   const displayAddress1 = address1
-    ? <>{address1}<br /></>
+    ? (
+      <>
+        {address1}
+        <br />
+      </>
+      )
     : null;
   const address2 = `${city}, ${state} ${zipCode}`;
+  const rouletteText =
+  !inRoulette.some(el => el.restaurantId === id) && !inRoulette.includes(id)
+    ? 'Add to Roulette'
+    : 'Remove from Roulette';
```

Comment:
This seems really inefficient, can we program the server to just give us an array of strings?